commit c8649cbe4d24ed3078151b56a8af9f3fce67b6db (refs/changes/79/4979/1)
Author: dyep <dyep49@gmail.com>
Date:   2018-10-19T12:00:46-07:00 (12 months ago)
    
    OS-7217 Snapshots of bhyve VMs should be mounted on state change

diff --git a/usr/src/lib/brand/bhyve/zone/statechange b/usr/src/lib/brand/bhyve/zone/statechange
index 76cfbffc17..398f46b4fd 100644
--- a/usr/src/lib/brand/bhyve/zone/statechange
+++ b/usr/src/lib/brand/bhyve/zone/statechange
@@ -15,6 +15,9 @@
 
 ps_brand=bhyve
 
+# Should we show snapshots?
+jst_showsnap="yes"
+
 typeset -A FORCED_ATTRS
 FORCED_ATTRS["zlog-mode"]=g--
 FORCED_ATTRS["zlog-name"]=platform.log
